lib/graphql.rb

```diff
@@ -101,8 +101,8 @@ module EmptyObjects
 require "graphql/pagination"
 require "graphql/schema"
 require "graphql/query"
-require "graphql/types"
 require "graphql/dataloader"
+require "graphql/types"
```

Comment:
I ran into an issue with `GraphQL::Dataloader::NullDataloader` not being defined at this point due to referencing `Directive` within `Scalar`